custom_components/ai_thermostat/models/utils.py

```diff
@@ -1,8 +1,6 @@
-import asyncio
 import logging
-from datetime import datetime, timedelta
```

Comment:
This is not part of the PR's intended changes. Please remove this from your changeset :)